cmd/aries-js-worker/src/aries.js

```diff
@@ -99,6 +99,24 @@ export const Aries = function(opts) {
         return INSTANCE
     }
 
+    if (!opts) {
+        throw new Error("aries: missing options")
+    }
+
+    if (!opts.assetsPath) {
+        throw new Error("aries: missing assets path")
+    }
+
+    WORKER = loadWorker(
+        PENDING,
+        NOTIFICATIONS,
+        {
+            dir: opts.assetsPath,
+            wasm: opts.assetsPath + "/aries-js-worker.wasm.gz",
```

Comment:
we should also allow configuration of whether or not the wasm was pre-compressed.
some servers prefer uncompressed and then they returned the compressed encoding as needed.

Reply:
new issue?

Reply:
sure.

(the other idea would be to allow customization of the filename including extension).

Reply:
#1277